catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -35,6 +35,8 @@ define([
             idAttribute: 'name',
             initialize: function () {
                 this.set('currentConfiguration', undefined);
+                this.set('currentUrl', undefined);
```

Comment:
We can avoid this initialize by using defaults.

```
defaults: function() {
   return {
    currentConfiguration: undefined,
    currentUrl:     undefined,
    currentWarning:   undefined,
    disabledConfigurations: new Source.ConfigurationList()
  };
}
```
Then again, I'm not familiar with this area of the code, so something might be relying on those sets to trigger changes.